Core/src/org/sleuthkit/autopsy/datasourcesummary/ui/Bundle.properties-MERGED

```diff
@@ -103,6 +103,17 @@ PastCasesPanel.notableFileLabel.text=Cases with Common Items That Were Tagged as
 PastCasesPanel.sameIdLabel.text=Past Cases with the Same Device IDs
 DataSourceSummaryTabbedPane.noDataSourceLabel.text=No data source has been selected.
 TimelinePanel.activityRangeLabel.text=Activity Range
+RecentFilesPanel.rightClickForMoreOptions1.text=Right click on row for more options
+RecentFilesPanel.rightClickForMoreOptions2.text=Right click on row for more options
+RecentFilesPanel.rightClickForMoreOptions3.text=Right click on row for more options
+UserActivityPanel.rightClickForMoreOptions1.text=Right click on row for more options
+UserActivityPanel.rightClickForMoreOptions2.text=Right click on row for more options
+UserActivityPanel.rightClickForMoreOptions3.text=Right click on row for more options
+UserActivityPanel.rightClickForMoreOptions4.text=Right click on row for more options
+UserActivityPanel.rightClickForMoreOptions5.text=Right click on row for more options
+TimelinePanel.viewInTimelineBtn.text=View in Timeline
+UserActivityPanel_goToArtifact=Go to Artifact
+UserActivityPanel_goToFile=Go to File
```

Comment:
We want to use the term Result in place of Artifact in text the end user will read.

Also we may want to consider the following phrasing instead for consistency with the rest of the application.
"View Source Result" and
"View Source File in Directory"